packages/hdwallet-core/src/bitcoin.ts

```diff
@@ -341,12 +341,6 @@ export interface BTCWalletInfo extends HDWalletInfo {
    */
   btcGetAccountPaths(msg: BTCGetAccountPaths): Array<BTCAccountPath>;
 
-  /**
-   * Does the device support spending from the combined accounts?
-   * The list is assumed to contain unique entries.
-   */
-  btcIsSameAccount(msg: Array<BTCAccountPath>): boolean;
```

Comment:
oh I see you removed it. major version bump change to interface? (no idea what the point of this function was for)

Reply:
Yeah given the fact that we have removed quite a big of packages, we should definitely go with at the very least a minor vs. a patch as we usually do. Not sure about major here, something something about open-source. 

Looking at Trezor implementation, this is used to detect diff. purposes over same slip44 + accountId... which we can do, and actually do pretty easily in web already! 
<img width="1031" alt="image" src="https://github.com/user-attachments/assets/bc4d26a3-76eb-40f9-b87e-14d35417e39c">